### PR TITLE
Add contributing  guidelines

### PR DESCRIPTION
**Pull request by jwu910**
Add contributing guidelines. https://github.com/5-gwoap/slng-node/issues/12

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+1. Fork this project
+2. Choose an issue
+3. Fix it!
+4. Send a PR to `development`
+
+#### Branch Names
+Try to keep branch names following the following format:
+
+  `slng-<issue number\>-topic`
```
Comment (rainrivas) [inline CONTRIBUTING.md]: Should this be `<issue number>-topic` and not have the extra `\`

File: README.md [hunk 2]
```diff
@@ -19,5 +19,13 @@ slng <search string>
 
 slng will return (up to) the first 3 results by default.
 
+
+#### Contributing
+1. Fork it
+2. Fix it
+3. PR it...?
```
Comment (rainrivas) [inline README.md]: Change this to something like "If you see an issue or think up a feature, open an issue on github. See our contribution guidelines <link to md>" or something

**Review by rainrivas — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (jwu910) [reply CONTRIBUTING.md]: Markdown was processing the <> and not rendering the text correctly w/o escaping the last >.

**Review by rainrivas — APPROVED** [review]

**Comment by jwu910** [comment]
🔥

**Pull request merged**